modules/proc_binding-operator-backed-components-by-adding-a-servicebinding-resource.adoc

```diff
@@ -0,0 +1,44 @@
+:_content-type: PROCEDURE
```

Comment:
@Dhruv-Soni11 If you intend to retain this file, then please add the [Module file metadata](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)

Reply:
Fixed. Deleting this file as it is no longer required.